src/vec_int64_ppc.h

```diff
@@ -471,7 +471,7 @@ vec_cmpgtud (vui64_t a, vui64_t b)
   /* Duplicate result word to dword width.  */
   y = vec_sld (c0, x, 12);
   r = vec_sel (x, y, c01);
-  result = (vui64_t)r;
+  result = (vb64_t)r;
```

Comment:
In a few other places, you removed a cast like this.  Do you need it here?

Reply:
No, in this case r and the vec_sel inputs are __vector unsigned int to work correctly on POWER7.